### ISSUE TITLE
Running `lerna-wizard` after install returns error

### ISSUE THREAD

**Issue by omonk** [szarouski/lerna-wizard#6]
I've installed lerna with `npm i lerna@prerelease -g` and `npm i lerna-wizard -g` as stated in the docs. Running `lerna-wizard` after install returns `env: node\r: No such file or directory`

**Comment by webuniverseio** [comment]
Hi @omonk. I'll look into your problem soon, and in meantime can you provide details about your setup: OS, version of node & npm you use, full error stack details, screenshot of the problem.

Thanks!

**Comment by omonk** [comment]
Hey @szarouski, of course 

macOS 10.12.2

Node: v6.9.4
npm: 3.10.10

No full error stack, just this one error

![image](https://cloud.githubusercontent.com/assets/7108120/23622585/2a917a4c-0297-11e7-9155-ef275b08d9e6.png)

**Comment by kristianmandrup** [comment]
I had the same problem. It's an OS linebreak error DOS to *Nix/Mac `/r` as [described here](https://github.com/nwjs/nw-builder/issues/75)

**Comment by kristianmandrup** [comment]
```bash
$ perl -MCwd -le 'print Cwd::abs_path(shift)' "/usr/local/bin/lerna-wizard"
/usr/local/lib/node_modules/lerna-wizard/bin/lerna-wizard

$ brew install dos2unix
$ sudo dos2unix /usr/local/lib/node_modules/lerna-wizard/bin/lerna-wizard
dos2unix: converting file /usr/local/lib/node_modules/lerna-wizard/bin/lerna-wizard to Unix format...

$ lerna-wizard
? What do you want to do? (Use arrow keys)
```

Sweet :)

**Comment by webuniverseio** [comment]
Thanks @kristianmandrup for looking into that. I reproduced the error on debian, so hopefully soon I'll fix it.

**Comment by webuniverseio** [comment]
@omonk , @kristianmandrup thanks for contribution. Problem should be resolved in 1.0.9. Please update and let me know if you run into issues.
